codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostRequestDecoderTest.java

```diff
@@ -546,7 +546,7 @@ public void testMultipartRequestWithFieldInvalidCharset() throws Exception {
     @Test
     public void testFormEncodeIncorrect() throws Exception {
         LastHttpContent content = new DefaultLastHttpContent(
-                Unpooled.copiedBuffer("project=netty&&project=netty", CharsetUtil.US_ASCII));
+                Unpooled.copiedBuffer("project=netty&=netty&project=netty", CharsetUtil.US_ASCII));
```

Comment:
The PR makes extraneous ampersands ignored. However if a parameter is defined with a key with no name it's still invalid; this test would fail because the new functionality means the previous test value is now no longer 'incorrect'.